<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Xingkai SC</family>
      <pointsize>27</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 200, 200) ;
color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yuanti SC</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 255, 130);

color: rgb(51, 15, 255);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>button_4</string>
    </property>
   </widget>
   <widget class="QWidget" name="background_wd" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <zorder>background_wd</zorder>
   <zorder>btn_1</zorder>
   <zorder>btn_2</zorder>
   <zorder>btn_3</zorder>
   <zorder>btn_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>1</string>
    </property>
    <widget class="QMenu" name="menu1_2">
     <property name="title">
      <string>2</string>
     </property>
     <widget class="QMenu" name="menu1_3">
      <property name="title">
       <string>3</string>
      </property>
      <addaction name="action1_4"/>
     </widget>
     <addaction name="menu1_3"/>
     <addaction name="action1_3"/>
    </widget>
    <addaction name="menu1_2"/>
   </widget>
   <addaction name="menu1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1_3">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_2</sender>
   <signal>clicked()</signal>
   <receiver>btn_1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
